py21cmsense/observation.py

```diff
@@ -121,6 +118,7 @@ class Observation:
 
     @classmethod
     def from_yaml(cls, yaml_file):
+        """Construct an :class:`Observation` from a YAML file."""
         if isinstance(yaml_file, str):
             with open(yaml_file) as fl:
                 data = yaml.load(fl, Loader=yaml.FullLoader)
```

Comment:
going to throw my 2-cents in here. FullLoader can be a dangerous (allows for arbitrary code execution). This isn't so bad if we can control where the yamls originate and trust the source though. Not saying it needs to go but I'm definitely weary of it, especially if a web-app is implemented. Do we want users to be providing this file at some point?

probably better to take this to a new issue, but wanted to get my thoughts down.

Reply:
#52